README.md

```diff
@@ -20,8 +20,40 @@ output {
         api_key => "<DATADOG_API_KEY>"
     }
 }
+```
+
+## Add metadata to your logs
+
+In order to get the best use out of your logs in Datadog, it is important to have the proper metadata associated with them (including hostname, service and source). 
+To add those to your logs, add them into your logs with a mutate filter:
 
 ```
+filter {
+  mutate {
+    add_field => {
+ "host" => "<HOST>"
```

Comment:
nits: `host => "<HOST>"` also leading white spaces are missing.
Same for the following lines